### PR TITLE
Add proxy installation status and translation support

### PR DESCRIPTION
**Pull request by stephdl**
This pull request includes changes to handle the installation status of the proxy module and display an info banner accordingly. It also adds translation support for the info banner.

image

https://github.com/NethServer/dev/issues/7057

### CODE REVIEW

File: imageroot/actions/get-configuration/20read [hunk 1]
```diff
@@ -20,6 +39,7 @@ config = {
     'reports_international_prefix': os.getenv('REPORTS_INTERNATIONAL_PREFIX', ''),
     'nethvoice_adm_username': os.getenv('NETHVOICE_USER_PORTAL_USERNAME', ''),
     'nethvoice_adm_password': agent.read_envfile("passwords.env").get('NETHVOICE_USER_PORTAL_PASSWORD',""),
+    "proxy_installed": proxy_installed,
```
Comment (Amygos) [inline imageroot/actions/get-configuration/20read:42]: The action `get-configuration` must return as output the same the same input of the `configure-module` action.
We need to find another way to get the proxy status.
Can be another action or extend the `configure-module` to accept the proxy moduleid/uuid

**Review by Amygos — CHANGES_REQUESTED** [review]

File: imageroot/actions/get-configuration/20read (hunk 1 above)
Comment (stephdl) [reply imageroot/actions/get-configuration/20read]: it should be good

**Comment by Amygos** [comment]
I let the review of the UI part to @edospadoni or @tonyco97

**Comment by edospadoni** [comment]
UI part is ok for me. Let's wait @tonyco97.

**Review by edospadoni — APPROVED** [review]

File: imageroot/actions/get-proxy-status/10get [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            # not configured
+            continue
+        module_node_id = rdb.hget(mkey, 'NODE_ID')
+        if module_node_id != node_id:
+            # not on this node
+            continue
+        else :
+            # finded
+            proxy_installed = True
+            break
```
[the remark this answers is not in the document]
Comment (stephdl) [reply imageroot/actions/get-proxy-status/10get]: I did it but @Amygos 

agent.resolve_agent_id('nethvoice-proxy@node') does not verify if the nethvoice proxy has been configured, once installed with  `module_public_ip == "127.0.0.1"` it returns `True`

my previous code was verifying that nethvoice proxy is installed and configured

[the remark this answers is not in the document]
Comment (Amygos) [reply imageroot/actions/get-proxy-status/10get]: You don't have to verify if is configured, the default configuration can be already used. When the admin sets the actual configuration, an event is raised the inform the NethVoice Module.

File: ui/public/i18n/en/translation.json [hunk 3]
```diff
@@ -71,7 +71,9 @@
     "error_message_hostname": "By changing the user provider of an already configured NethVoice, all the users and extensions configuration will be reset.",
     "nethvoice_host_format": "Must be a valid domain name be a valid domain name.",
     "nethcti_ui_host_format": "Must be a valid domain name be a valid domain name.",
-    "set_rebranding": "Rebranding setup"
+    "set_rebranding": "Rebranding setup",
+    "proxy_not_installed": "Nethvoice Proxy",
+    "proxy_not_installed_description": "The proxy needs to be installed and configured before configuring NethVoice."
```
Comment (Amygos) [inline ui/public/i18n/en/translation.json:76]:
```suggestion
    "proxy_not_installed_description": "The proxy needs to be installed before configuring NethVoice."
```

**Review by Amygos — APPROVED** [review]

File: ui/public/i18n/en/translation.json (hunk 3 above)
Comment (tonyco97) [reply ui/public/i18n/en/translation.json:76]: It's better to make all the translations using weblate, and not in this PR?

**Pull request merged**